app/components/autocomplete/slash_suggestion/app_slash_suggestion/app_slash_suggestion.tsx

```diff
@@ -0,0 +1,238 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import {intlShape} from 'react-intl';
+import React, {PureComponent} from 'react';
+import {
+    FlatList,
+    Platform,
+} from 'react-native';
+
+import {analytics} from '@init/analytics';
+import {Client4} from '@client/rest';
+import {isMinimumServerVersion} from '@mm-redux/utils/helpers';
+import {AutocompleteSuggestion} from '@mm-redux/types/integrations';
+import {Theme} from '@mm-redux/types/preferences';
+import {makeStyleSheetFromTheme} from '@utils/theme';
+
+import SlashSuggestionItem from '../slash_suggestion_item';
+import {AppCommandParser} from '../app_command_parser/app_command_parser';
+import {COMMAND_SUGGESTION_CHANNEL, COMMAND_SUGGESTION_USER, ExtendedAutocompleteSuggestion} from '../app_command_parser/app_command_parser_dependencies';
+import AtMentionItem from '@components/autocomplete/at_mention_item';
+import ChannelMentionItem from '@components/autocomplete/channel_mention_item';
+
+export type Props = {
+    currentTeamId: string;
+    isSearch?: boolean;
+    maxListHeight?: number;
+    theme: Theme;
+    onChangeText: (text: string) => void;
+    onResultCountChange: (count: number) => void;
+    value: string;
+    nestedScrollEnabled?: boolean;
+    rootId?: string;
+    channelId: string;
+    appsEnabled: boolean;
+};
+
+type State = {
+    active: boolean;
+    dataSource: AutocompleteSuggestion[];
+}
+
+export default class AppSlashSuggestion extends PureComponent<Props, State> {
+    static defaultProps = {
+        defaultChannel: {},
+        value: '',
+    };
+
+    static contextTypes = {
+        intl: intlShape.isRequired,
+    };
+
+    appCommandParser: AppCommandParser;
+
+    state = {
+        active: false,
+        dataSource: [],
+    };
+
+    constructor(props: Props, context: any) {
+        super(props);
+        this.appCommandParser = new AppCommandParser(null, context.intl, props.channelId, props.currentTeamId, props.rootId);
+    }
+
+    setActive(active: boolean) {
+        this.setState({active});
+    }
+
+    componentDidUpdate(prevProps: Props) {
+        if (!isMinimumServerVersion(Client4.getServerVersion(), 5, 24) || !this.props.appsEnabled) {
```

Comment:
the minimum supported version is 5.31 at this time, so we can just remove the check for the server version here